plugins/ssr/ssrvernac.mlg

```diff
@@ -309,9 +309,12 @@ END
       ~category:"deprecated" ~default:CWarnings.Enabled
       (fun () ->
          (Pp.strbrk
-            "SSReflect's Search command has been moved to the \
-             ssrsearch module; please Require that module if you \
-             still want to use SSReflect's Search command"))
+            "The Search command has been changed to refer to Coq's own \
+             search feature (formerly SearchAbout).\
+             In previous versions of Coq, the Search command was provided \
+             by SSreflect.\
+             If you want to use SSReflect's deprecated Search command \
+             instead of the built-in one, please import the ssrsearch module."))
```

Comment:
The old text said Require (note also the case), not import. I assume that was intentional (and hopefully accurate). Should it be kept?